onnx/test/shape_inference_test.py

```diff
@@ -976,6 +976,21 @@ def _rnn_bidirectional(self, seqlen, batchsize, inpsize, hiddensize):  # type: (
             make_tensor_value_info('all', TensorProto.FLOAT, (seqlen, 2, batchsize, hiddensize)),
             make_tensor_value_info('last', TensorProto.FLOAT, (2, batchsize, hiddensize))])
 
+    def test_rnn_batchwise(self):  # type: () -> None
+        self._rnn_batchwise(64, 32, 10, 4)
+
+    def _rnn_batchwise(self, seqlen, batchsize, inpsize, hiddensize):  # type: (int, int, int, int) -> None
```

Comment:
Maybe I missed it, but is there a shape inference test added for the bidirectional case with the new flag? If not, it would be nice to add one.

Reply:
I've added it